DOC-865: 5.9.0 suggested changelog edits (#7151)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- When scrolling, the context toolbar will stick to where it was previously for large elements, such as tables #TINY-7545
- The context toolbar will now move out of the way when it overlaps with the selection, such as in table cells #TINY-7192
- The context toolbar will now use transition animations when changing placements #TINY-7740
- The `formatter.match` API can now take an optional `similar` parameter to check if the format partially matches #TINY-7712
- `Env.browser` now uses the User-Agent Client Hints API when it is available #TINY-7785
- Icons with a `-rtl` suffix in their name will now automatically be used when the UI is rendered in right-to-left mode #TINY-7782
- `editor.formatter.formatChanged` now supports listening for changes to formats with specific variables #TINY-7713
- The `autolink` plugin link detection now permits custom protocols and improves valid link detection #TINY-7714

### Changed

> - Changed the load order so that the content CSS is loaded before the editor gets populated with contents #TINY-7249

Reviewer: ```suggestion
- Changed the load order so that the content CSS is loaded before the editor gets populated with content #TINY-7249
```
Should it be just content singular?

- HTML comments with mismatched quotes were parsed incorrectly under certain circumstances #TINY-7589
- Links in notification text did not show the correct mouse pointer #TINY-7661
- Applying selector formats would sometimes not apply the format correctly to elements in a list #TINY-7393
- For formats that specify an attribute or style that should be removed, the formatter `match` API incorrectly returned `false` #TINY-6149
- The type signature on `editor.formatter.matchNode` had the wrong return type (was `boolean` but should have been `Formatter | undefined`) #TINY-6149
- Menus and context menus were not closed when clicking into a different editor #TINY-7399

> - Using the Tab key to navigate into the editor on IE 11 would incorrectly focus the toolbar #TINY-3707

Reviewer 2: ```suggestion
- Using the Tab key to navigate into the editor on Microsoft Internet Explorer 11 would incorrectly focus the toolbar #TINY-3707
```

- The editor selection could be placed in an incorrect location when undoing or redoing changes in a document containing `contenteditable="false"` elements #TINY-7663
- Context menus on Android were not displayed when more than one HTML element was selected #TINY-7688

> - Certain HTML content, when inserted could cause the editor to crash #TINY-7756

Reviewer 2: ```suggestion
- The editor could crash when inserting certain HTML content #TINY-7756
```
